src/libData/AccountData/AccountStore.cpp

```diff
@@ -291,10 +291,12 @@ void AccountStore::RevertCommitTemp() {
 
   // Revert changed
   for (auto const entry : m_addressToAccountRevChanged) {
+    // LOG_GENERAL(INFO, "Revert changed address: " << entry.first);
```

Comment:
Do you want to keep this commented-out log message?

Reply:
same as the below one